lib/src/document_init_parameters.dart

```diff
@@ -97,6 +101,11 @@ class DocumentInitParameters {
     _jsInternal?['withCredentials'] = withCredentials;
   }
 
+  bool? get isEvalSupported => _jsInternal?['isEvalSupported'] as bool?;
+  set isEvalSupported(bool? isEvalSupported) {
+    _jsInternal?['isEvalSupported'] = isEvalSupported;
+  }
```

Comment:
🤔 Should we remove the public setter?

Reply:
Yeah, I'd probably do that since we def don't want this being set by anyone